packages/communication-react/review/communication-react.api.md

```diff
@@ -1240,13 +1237,6 @@ export interface MicrophoneButtonStrings {
     onLabel: string;
 }
 
-// @public
-export const namedLocales: Record<string, {
```

Comment:
This was an alternative export for the named exports `en_US` etc.
Removed because:
* Redundant
* Not described in storybook
* Actually works badly with tree-shaking -- customers would get all language bundles by pulling this symbol.

Reply:
Turns out this is used in the storybook to populate the list of available locales.

TODO: Drop this use in storybook, instead just list the locales there.

Reply:
Done